scripts/README.md

```diff
@@ -29,17 +42,8 @@ The following steps must be proposed as a pull request.
 
 ### Documentation
 
-Push the next branch on the release branch to deploy the documentation with the latest changes. It lives at https://material-ui.netlify.app/. Force push if necessary.
-
-Note: if you don't have the `material-ui-docs` remote already, you should add it with
-
-```sh
-git remote add material-ui-docs https://github.com/mui/material-ui-docs.git
-```
-
-```sh
-yarn docs:deploy
-```
+`yarn docs:deploy` to deploy the documentation (it lives at https://material-ui.netlify.app/) with the latest changes.
+Force push if necessary.
```

Comment:
We could almost add `-f` in

https://github.com/mui/material-ui/blob/16a8adb01c093ac33ff178032bfda52b493ed615/docs/package.json#L12

So we never need to worry about this. It's totally normal to have to push force in the current workflow.

```suggestion
```

Reply:
I also wondered if this was more convenient – is it common to have to force push to get this part to work? @mui/core

Reply:
It depends on either website changes were cherry-picked to be deployed outside of the MUI Core docs release cycle. Maybe 5% of the time?

Reply:
I didn't needed to force push for a long time, but having it by default would indeed improve the experience, especially for new joiners 👍

Reply:
Yeah, we had to cherry-pick to do a hot deploy to fix https://mui-org.slack.com/archives/C041SDSF32L/p1676634426113669.